.github/POLICYCONFIG.md

```diff
@@ -0,0 +1,34 @@
+Welcome to this short guide to the POLICY config mechanism.
+
+You are probably reading this guide because you have been informed your antagonist or ghost role needs to support policy configuration.
+
+## Requirements
+It is a requirement of /tg/station development that all ghost roles, antags, minor antags and event mobs of any kind must support the policy system when implemented.
+
+## What is policy configuration
+Policy configuration is a json file that the administrators of a server can edit, which contains a dictionary of keywords -> string message.
+
+The policy text for a specific keyword should be displayed when relevant and appropriate, to allow server administrators to define the broad strokes of policy for some feature or mob.
+
+It is okay to provide a default text when the config is not set, but you are required to provide the config in all cases of a ghost role or an antagonist, or minor event.
+
+If you're in doubt about needing to support policy config, I suggest doing it anyway. This should replace all flavour text, ghost spawn messages and so forth that the player (i.e client) sees upon entering the role, mob, feature.
```

Comment:
```suggestion
If you're in doubt about needing to support policy config, I suggest doing it anyway. This should replace all flavour text, ghost spawn messages and so forth that the player (i.e client) sees upon entering the role, mob, feature that is meant to dictate how they are permitted to be played as/with.
```
https://github.com/tgstation/tgstation/pull/50185#issuecomment-603876333

Reply:
that's not gramatically correct now dip ass, it should say role, mob or feature

Reply:
sorry english is my first language!